src/platform/packages/shared/kbn-alerts-ui-shared/src/alert_filter_controls/alert_filter_controls.tsx

```diff
@@ -120,8 +125,14 @@ export const AlertFilterControls = (props: AlertFilterControlsProps) => {
       }
     }
 
-    return () => dataViews.clearInstanceCache();
-  }, [dataView, dataViewSpec, dataViews, isLoadingDataView]);
+    return () => {
+      if (disableCacheManagement) {
```

Comment:
nit: If this prop won't do anything else, we might want to go for something more specific like `preventCacheClearOnUnmount`. It would avoid suggesting broader cache management responsibilities

Reply:
good suggestion, applied 👌